website/docs/d/public_ips.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_public_ips"
+sidebar_current: "docs-azurerm-datasource-public-ips"
+description: |-
+  Provides a list of public IP addresses.
+---
+
+# azurerm_public_ips
+
+Use this data source to access a filtered list of Public IP Addresses
+
+## Example Usage
+
+```hcl
+data "azurerm_public_ips" "test" {
+  resource_group_name = "pip-test"
+  attached            = false
+}
+```
+
+## Argument Reference
+
+* `resource_group_name` - (Required) Specifies the name of the resource group.
+* `attached` - (Optional) Should we only return IP Addresses which are attached to a device (VM/LB) etc?
```

Comment:
I wouldn't frame this as a question and instead have it read like `Returns IP Address that are attached to a device or not attached to a device` or something to that effect.